### ISSUE TITLE
OpenCon Zine :O

### ISSUE THREAD

**Issue by JosephMcArthur** [sparcopen/opencon#104]
Suggested in #80 by @pennybphd to an amazing reception. 

Would love ideas on how to make this happen.

**Comment by RaoOfPhysics** [comment]
Do we want to move this discussion further?

**Comment by char-siuu-bao** [comment]
@RaoOfPhysics We discussed this on a staff call and decided it might be a little hard to implement in practice — that being said .... I just remembered one of our participants, Linda Lew is a zine expert!! I believe she used to run the Aukland Zinefest  (and I'm planning to do a zine swap with her while we are in Berlin!) - we could see if she's up for informally facilitating something, but I'm not sure we should sink too many resources into this.

**Comment by pennybphd** [comment]
Maybe a small group of us can doathon it?

**Comment by char-siuu-bao** [comment]
@pennybphd, yes that could definitely potentially work!! If a zine was a do-a-thon output, I would love for it to have a clear goal and purpose so it could be usefully re-distributed in the open community (e.g. maybe one built off of DEI conversations could be super awesome!!)
